### PR TITLE
Adds delete node code snippets to the cheatsheet.md

### PR DESCRIPTION
**Pull request by Stockfoot**
## Description
Related Drupal Issue(s): [Add to Cheatsheet: User/User Role can delete a piece of content](https://www.drupal.org/project/shrubs/issues/3402918)

**This PR adds 2 items to the cheatsheet**
- Delete content from node page
- Delete content from node edit page

### CODE REVIEW

File: CHEATSHEET.md [hunk 1]
```diff
@@ -215,3 +219,33 @@ cy.createPage();
...
 cy.logout();
 ```
+
+### Delete a piece of content from the node page
+
+```markdown
+// Click node Delete button.
+cy.get('.nav-link').contains('Delete').click();
+// Click confirm delete.
+cy.get('.button--danger').click();
```
Comment (thejimbirch) [inline CHEATSHEET.md]: Can we change this to `.contains('Delete')` also?  `.button--danger` may not be on all admin themes.

File: CHEATSHEET.md [hunk 2]
```diff
@@ -215,3 +219,33 @@ cy.createPage();
 cy.createPerson();
 cy.logout();
 ```
+
+### Delete a piece of content from the node page
+
+```markdown
```
Comment (thejimbirch) [inline CHEATSHEET.md:227]: We need a visit command here first.

// Visit a node
cy.visit('node/[nodeID]')

File: CHEATSHEET.md [hunk 3]
```diff
@@ -215,3 +219,33 @@ cy.createPage();
...
+// Click node Delete button.
+cy.get('.nav-link').contains('Delete').click();
+// Click confirm delete.
+cy.get('.button--danger').click();
+// Validate, we should be on the home page.
+cy.get('h1').should('contain', 'Hamilton Educational Website');
```
Comment (thejimbirch) [inline CHEATSHEET.md]: This is too specific.  Is there another way we can validate the deletion?  If not, please update this to say something like `Your Page Title`.

File: CHEATSHEET.md [hunk 4]
```diff
@@ -215,3 +219,33 @@ cy.createPage();
...
+const deleteModalAjax = 'deleteModalAjax' + '#edit-delete--2' + Math.random();
+cy.intercept('POST', '/media-library**').as(deleteModalAjax)
+// Get the delete modal and delete the node.
+cy.get('.ui-dialog-buttonpane').within(($modal) => {
+  // Get the delete button in the modal and click.
+  cy.get('.button--danger').contains('Delete', { matchCase: false }).click();
+})
+// Validate, we should be on the admin/content page.
+cy.get('h1').should('contain', 'Content');
+```
```
Comment (thejimbirch) [inline CHEATSHEET.md:258]: Awesome!

**Review by thejimbirch — CHANGES_REQUESTED** [review]

**Comment by paulsheldrake** [comment]
There may be a new support function here...  something like

```
deleteNode($title) {
visit(/admin/content)
search filter for title
click delete in the options
check yes
}
```

it would require that user have permissions to the content overview page

**Review by thejimbirch — APPROVED** [review]

**Pull request merged**